app/utils/RegimeHelper.scala

```diff
@@ -47,4 +47,6 @@ object RegimeHelper {
       case _ => ""
     }
   }
+
+  def constructMTDVATEnrolmentKey(vrn: String): String = s"HMRC-MTD-VAT~VRN~$vrn"
```

Comment:
As in the  schema the  pattern for VRN is "^([a-zA-Z0-9]{1,20})$", do this need to be validated against this pattern

Reply:
Do we not need a test case for this method.

Reply:
Checking the need for validation now...

Will add tests cases - I didn't add them before due to the lack of tests for the helper, but worth adding them

Reply:
Pushed some tests, waiting on an answer for validation

Reply:
Validation should be performed - I will validate it against: https://github.com/hmrc/service-enrolment-config/blob/d95a95f054c1638049677f69c7952b8ff00ac2e7/conf/SEC1_with_enrolment_rules_json/prod/HMRC-MTD-VAT.json#L7